tutorials/shaders/your_first_shader/your_first_2d_shader.rst

```diff
@@ -132,27 +132,27 @@ other functions or to assign values to ``COLOR`` directly.
 Using ``TEXTURE`` built-in
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-When you want to adjust a color in a Sprite2D you cannot just adjust the color
+The default fragment function reads from the set Sprite2D texture and displays it.
+
+When you want to adjust a color in a Sprite2D you can adjust the color
 from the texture manually like in the code below.
 
 .. code-block:: glsl
 
   void fragment(){
-    //this shader will result in an all white rectangle
+    //this shader will result in a blue-tinted icon
```

Comment:
```suggestion
    // This shader will result in a blue-tinted icon.
```
Since we're fixing things here I'd say to fix the comment style while we're at it, for the whole file (we're changing 1/3 of the comments anyway)